### PR TITLE
Fix for #2433

### PR DESCRIPTION
**Pull request by leftbrasil**
# PR Details

## Description

Fix for #2433 

## Related Issue

#2433
## How Has This Been Tested

Tested on my own project 

## Types of changes

- [ ] Dependency change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x ] My code follows the code style of this project.
- [ NA] I have updated the documentation accordingly (index.d.ts).

### CODE REVIEW

**Review by pedroherpeto — APPROVED** [review]

**Review by vinibgoulart — COMMENTED** [review]
I'm not sure if this is the right way, what will it impact?

**Comment by marcosbozza** [comment]
Apparently they just replaced `data-testid` with `data-icon`.

image

It could just be changed to the following to keep selecting the intro image.

```
const INTRO_IMG_SELECTOR = '[data-icon="intro-md-beta-logo-dark"], [data-icon="intro-md-beta-logo-light"], [data-asset-intro-image-light="true"], [data-asset-intro-image-dark="true"]';
```

The suggested change `[data-icon="chat"]` is selecting the new chat button, which I imagine will not go away and it also means that the user is authenticated.

image

So, I believe the impact is small to change to `[data-icon="chat"]`.

**Review by marcosbozza — APPROVED** [review]

File: src/Client.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ class Client extends EventEmitter {
             }
         );
 
-        const INTRO_IMG_SELECTOR = '[data-testid="intro-md-beta-logo-dark"], [data-testid="intro-md-beta-logo-light"], [data-asset-intro-image-light="true"], [data-asset-intro-image-dark="true"]';
+        const INTRO_IMG_SELECTOR = '[data-icon=\'chat\']';
```
Comment (shirser121) [inline src/Client.js:175]: please change to `'[data-icon="chat"]'`

Except this, looking good

**Review by shirser121 — APPROVED** [review]
LGTM
Checked it also on older version

**Review by tuyuribr — APPROVED** [review]

**Comment by manoedinata** [comment]
I applied the patch, but it isn't working.
I tried to inspect WhatsApp Web's element, and I found that it is now `data-icon='new-chat'`

image

Changing it to `new-chat` works for me.

```javascript
const INTRO_IMG_SELECTOR = '[data-icon=\'new-chat\']';
```

**Pull request merged**